modules/prebidServerBidAdapter.js

```diff
@@ -19,15 +19,39 @@ const DEFAULT_S2S_CURRENCY = 'USD';
 const DEFAULT_S2S_NETREVENUE = true;
 
 let _s2sConfig;
+
+const s2sDefaultConfig = {
+  enabled: false,
+  timeout: 1000,
+  maxBids: 1,
+  adapter: 'prebidServer'
+};
+
 config.setDefaults({
-  's2sConfig': {
-    enabled: false,
-    timeout: 1000,
-    maxBids: 1,
-    adapter: 'prebidServer'
-  }
+  's2sConfig': s2sDefaultConfig
 });
 
+// accountId and bidders params are not included here, should be configured by end-user
+const availVendorDefaults = {
+  'appnexus': {
+    adapter: 'prebidServer',
+    cookieSet: false,
+    enabled: true,
+    endpoint: '//prebid.adnxs.com/pbs/v1/auction',
+    syncEndpoint: '//prebid.adnxs.com/pbs/v1/cookie_sync',
+    timeout: 1000
+  },
+  'rubicon': {
+    adapter: 'prebidServer',
+    cookieSet: true,
+    cookieSetUrl: 'https://secure-assets.rubiconproject.com/utils/cookieset/cs.js',
+    enabled: true,
+    endpoint: 'https://prebid-server.rubiconproject.com/auction',
```

Comment:
@bretg 
You want these hardcoded to HTTPS right?

Reply:
We've been avoiding the protocol-relative URLs due to articles like

https://stackoverflow.com/questions/28446314/why-use-protocol-relative-urls-at-all
https://jeremywagner.me/blog/stop-using-the-protocol-relative-url/

You haven't seen issues with protocol-relative URLs?

In any case, go ahead with this for now and we'll start the conversation internally.

Reply:
Talked to @mkendall07 -- the issue here is that non-secure demand is left on the table for HTTPS.

So, yes, let's switch the endpoints to protocol-relative. Thanks Matt. We'll take this discussion up internally as well.